app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class UsersController < Api::ApplicationController
```

Comment:
Also it is good to restrict your controllers regarding requested format. In your case allow only json.